packages/react/src/components/BarChartCard/BarChartCard.jsx

```diff
@@ -363,6 +366,8 @@ const BarChartCard = ({
       isResizable={isResizable}
       resizeHandles={resizeHandles}
       timeRange={timeRange}
+      // TODO: remove deprecated testID in v3.
```

Comment:
Yeah, generally I agree, but there are so many of them that it was much easier to create in-code todos--especially since we'll start the v3 branch and remove them within the next month anyway.